tools/astarte_dev_tool/lib/utilities/process.ex

```diff
@@ -0,0 +1,40 @@
+defmodule AstarteDevTool.Utilities.Process do
+  def check_process(command, args, path) when is_list(args) do
+    check_process(command <> " " <> Enum.join(args, " "), path)
```

Comment:
The call is somewhat hard to parse. I suggest
```suggestion
    command = "#{command} #{Enum.join(args, "")}"
    check_process(command, path)
```